scripts/crawl.js

```diff
@@ -16,6 +16,8 @@ const excludedUrls = [
   'wiki.jenkins.io', // Slow response
   'www.rubydoc.info', // Slow response
   'github.com', // managed as failed
+  'twitter.com',
+  'hamcrest.org',
```

Comment:
should we instead update/remote all invalid links to hamcrest docs?

Reply:
Will fix the hamcrest thing. https://www.packtpub.com/ also slow response, so it should also be added here for now.

Reply:
It seems hamcrest.org itself down. We can add this here so far, I think